### PR TITLE
remove count from Model.php

### PR DESCRIPTION
**Pull request by badrshs**
no need to use count here
fix error for php >7
 count(): Parameter must be an array or an object that implements Countable.

you can remove `count `from this and the value gonna be same

 ```
public function hasCustomPrimaryKey()
    {
        return count($this->primaryKeys->columns) == 1 &&
               $this->getPrimaryKey() != $this->getDefaultPrimaryKeyField();
    }
```

### CODE REVIEW

File: src/Coders/Model/Model.php [hunk 1]
```diff
@@ -745,7 +745,7 @@ public function getConnectionName()
      */
     public function hasCustomPrimaryKey()
     {
-        return count($this->primaryKeys->columns) == 1 &&
+        return ($this->primaryKeys->columns) == 1 &&
```
Comment (CristianLlanos) [inline src/Coders/Model/Model.php]: What does this supposedly do?

Comment (camiloclc) [reply src/Coders/Model/Model.php]: I had this same problem, an Exceptino was thrown: `ErrorException  : count(): Parameter must be an array or an object that implements Countable`. Turns out there wasn't a primary key defined in one of my tables...

Comment (CristianLlanos) [reply src/Coders/Model/Model.php]: That's helpful. Thanks!

Comment (myusuf5400) [reply src/Coders/Model/Model.php]: how about check if the variable of $this->primaryKeys->columns is array?

**Comment by CristianLlanos** [comment]
I'm closing this one in favor of #93

**Pull request closed without merging**